content/en/docs/ambient/architecture/index.md

```diff
@@ -9,57 +9,109 @@ owner: istio/wg-networking-maintainers
+  - If a namespace is labeled with `istio.io/use-waypoint` with its default waypoint for the namespace, the waypoint will apply to all pods in the namespace.
+  - The `istio.io/use-waypoint` label can also be set on individual services or pods when using a waypoint for the entire namespace is not desired.
+  - If the `istio.io/use-waypoint` label exists on both a namespace and a service, the service waypoint takes
+  precedence over the namespace waypoint as long as the service waypoint can handle service or all traffic.
+  Similarly, a label on a pod will take precedence over a namespace label.
 
 Depending on which category a workload is in, the request path will be different.
 
 ### Ztunnel routing
 
 #### Outbound
 
-When a captured pod makes an outbound request, it will be transparently redirected to ztunnel which will determine where and how to forward the request.
+When a pod in ambient mesh makes an outbound request, it will be transparently redirected to ztunnel which will determine where and how to forward the request.
 In general, the traffic routing behaves just like Kubernetes default traffic routing;
 requests to a `Service` will be sent to an endpoint within the `Service` while requests directly to a `Pod` IP will go directly to that IP.
 
 However, depending on the destination's capabilities, different behavior will occur.
-If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted {{< gloss "HBONE" >}}HBONE tunnel{{< /gloss >}}.
-If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will instead be forwarded to that waypoint.
+If the destination is also added in the mesh, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted {{< gloss "HBONE" >}}HBONE tunnel{{< /gloss >}}.
+If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will also be forwarded to that waypoint for L7 policy enforcement.
 
-Note that in the case of a request to a `Service`, a specific endpoint will be selected to determine if it has a waypoint.
-However, if it *has* a waypoint, the request will be sent with a target destination of the `Service`, not the selected endpoint.
-This allows the waypoint to apply service-oriented policies to the traffic.
-In the rare case that a `Service` has a mix of waypoint enabled and non-enabled endpoints, some requests would be sent to a waypoint while other requests to the same service would not.
+Note that in the case of a request to a `Service`, if the service *has* a waypoint, the request will be sent to its waypoint to enforce L7 policies to the traffic.
+Similarly, in the case of a request to a `Pod` IP, if the pod *has* a waypoint, the request will be sent to its waypoint to enforce L7 policies to the traffic.
```

Comment:
@bleggett i think you had a comment earlier to clarify if the use-waypoint label for both service and pod, what happens... It is explained here imho as it is for outbound, not inbound.

Reply:
Yep fair enough - we could make it a bit more explicit but this will do.